crates/uv-virtualenv/src/bare.rs

```diff
@@ -179,27 +189,126 @@ pub fn create_bare_venv(
+                                                        .join("nt")
+                                                        .join(launcher)
+                                                        .user_display(),
+                                                    base_python.with_file_name(launcher).user_display(),
+                                                    base_python
+                                                        .with_file_name(python_exe)
+                                                        .user_display(),
+                                                 )
+                                            )));
+                                        }
+                                        Err(err) => {
+                                            return Err(err.into());
+                                        }
+                                    }
+
+                                    // Copy `.dll` and `.pyd` files from the top-level, and from the
+                                    // `DLLs` subdirectory (if it exists).
+                                    for directory in [
+                                        python_home,
+                                        interpreter.base_prefix().join("DLLs").as_path(),
+                                    ] {
+                                        let entries = match fs_err::read_dir(directory) {
+                                            Ok(read_dir) => read_dir,
+                                            Err(err) if err.kind() == io::ErrorKind::NotFound => {
+                                                continue;
+                                            }
+                                            Err(err) => {
+                                                return Err(err.into());
+                                            }
+                                        };
+                                        for entry in entries {
+                                            let entry = entry?;
+                                            let path = entry.path();
+                                            if path.extension().is_some_and(|ext| {
+                                                ext.eq_ignore_ascii_case("dll")
+                                                    || ext.eq_ignore_ascii_case("pyd")
+                                            }) {
+                                                if let Some(file_name) = path.file_name() {
+                                                    fs_err::copy(&path, scripts.join(file_name))?;
+                                                }
+                                            }
+                                        }
+                                    }
+
+                                    // Copy `.zip` files from the top-level.
+                                    let entries = match fs_err::read_dir(python_home) {
```

Comment:
Virtualenv is stricter on what it matches here, but I don't see other `.zip` files anyway?